packages/builder/plugin-swc/package.json

```diff
@@ -64,7 +64,7 @@
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.21.5",
     "@modern-js/builder-shared": "workspace:*",
-    "@modern-js/swc-plugins": "0.5.5",
+    "@modern-js/swc-plugins": "0.6.0",
```

Comment:
Is `react-const-elements` enabled by default?

Reply:
We do not enable it when using babel, and the default behavior of SWC should be same as babel.

Reply:
No, it's not enabled by default

Reply:
get